api/filter/accesslog.proto

```diff
@@ -0,0 +1,110 @@
+syntax = "proto3";
+
+package envoy.api.v2.filter;
```

Comment:
As an aside, I'm hoping we can really stabilize moving things between packages. There are a lot of folks seriously looking at v2, the more churn at the API level, the less seriously folks will take our statement that v2 is ready and stable. Maybe we should do one last mega sweep after this of anything that needs to be moved and then declare a freeze, for realsy. This comment is non-actionable for this PR.

Reply:
Maybe we should work together to clarify what frozen means? From my perspective it's wire frozen. Refactoring is a pain in the ass for people who need to fix code but is still wire compatible? (I agree with your sentiment, but we should be clear what it means).

Reply:
Agreed, but we also just made a wire change in this PR :cricket: 

Thanks for pointing out this wire/API distinction. I think we want both wire and API freezing before we can say we're "stable"; folks working with the Java SDK (for example) don't expect classes to jump between packages arbitrarily between released versions, we should try and provide similar guarantees at some I think. WDYT?

Reply:
@htuch I'm torn on the code movement. To me it's not much different from changing something in the filter API. So perhaps it should be more of a deprecation thing? Like we could move stuff and then delete it in the next release? I'm fine either way though if for protos we want to be stricter about this.

On the wire change, @zuercher, sorry to be a mega PITA, but can you revert that part of the change (put it back)? I think we have come full circle and want that config option, so now we aren't doing a breaking wire change. :)